testing/infra/machines.yml

```diff
@@ -1,12 +1,20 @@
 ---
 
+fedora30:
+ type: image
+ pool: image_pool
+
+image_pool:
+  type: pool
+  path: /srv/
+
 vm_net:
  type: network
  cidr: 10.0.5.0/24
 
 micro_fedora:
   type: profile
-  image: Fedora-Cloud-Base-30-1.2.x86_64.qcow2
+  image: fedora30
```

Comment:
That convenience mechanism actually never worked for me! This long form has a reason.

Reply:
@karmab please, confirm. Not certain is it down to when the image storage sits on pools, but doubt.

Reply:
yeah, i fixed the convenience stuff some weeks ago